packages/block-editor/src/components/link-control/style.scss

```diff
@@ -462,13 +462,21 @@ $preview-image-height: 140px;
 	}
 }
 
-.block-editor-link-control__tools {
+.block-editor-link-control__tools,
+.block-editor-link-control__tools_in_preview {
 	display: flex;
-	flex-wrap: wrap;
 	align-items: center;
-	justify-content: space-between;
 	margin: 0;
 	padding: $grid-unit-20;
+}
+
+.block-editor-link-control__tools_in_preview {
+	border-top: 1px solid #ddd;
+}
+
+.block-editor-link-control__tools {
+	flex-wrap: wrap;
+	justify-content: space-between;
```

Comment:
Thank you for looking at visual design here. However, it's really important we try to keep PRs small and focused. This makes it easier to roll back changes that cause regressions and also makes it easier to review PRs.

I think we should remove everything from this PR that doesn't fix [the bug ](https://github.com/WordPress/gutenberg/issues/45741). 

Feel free to spin out other PRs with the other changes.

Apologies if this seems annoying or pedantic but it's proven to be an important principle of working in a large open source codebase. Thank you 🙇‍♂️